skunk.gemspec

```diff
@@ -14,6 +14,8 @@ Gem::Specification.new do |spec|
   spec.description   = "Knows how to calculate the StinkScore for a Ruby file"
   spec.homepage      = "https://github.com/fastruby/skunk"
 
+  spec.required_ruby_version = ">= 2.5.0"
```

Comment:
Does the test suite breaks using `2.0.0`?

Reply:
@bronzdoc Not sure, but we're only testing with Ruby 2.5 and above. 

I'm trying to find out if it works over here: https://github.com/fastruby/skunk/pull/38/commits/5f42de978d0e7b610c6cf58df0a87bea9de7012e

Reply:
@bronzdoc Actually, trying to answer your question: https://github.com/fastruby/skunk/pull/39

Reply:
@etagwerker I think the highest we should support is ruby >= 2.4. I updated https://github.com/fastruby/skunk/pull/39